### PR TITLE
Modify wrong comment

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

Modify wrong comment

Thanks!

**Special notes for your reviewer**:

**Release note**:

```release-note
```

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37463)

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

**Comment by thockin** [comment]
@caesarxuchao Are we editing files in staging/ or in their "real" home only?

**Comment by caesarxuchao** [comment]
In their real homes. The changes to the staging area will be overwritten once someone runs [copy.sh](https://github.com/kubernetes/kubernetes/blob/master/staging/copy.sh).

File: pkg/client/cache/listers.go [hunk 1]
```diff
@@ -466,7 +466,7 @@ func (s *storageClassLister) List(selector labels.Selector) (ret []*storage.Stor
 	return ret, err
 }
 
-// List returns a list of storage classes
+// Get returns storage.StorageClass for the StorageClass 'name'
```
Comment (caesarxuchao) [inline pkg/client/cache/listers.go]: `Get returns storage class with name "name"` sounds more accurate.

**Comment by caesarxuchao** [comment]
lgtm. Thank you @tanshanshan.

**Comment by rmmh** [comment]
@k8s-bot gci gke e2e test this
@k8s-bot verify test this

**Comment by rmmh** [comment]
@k8s-bot node e2e test this

**Pull request merged**